tests/pytest_tests/system_tests/test_public_api.py

```diff
@@ -58,6 +58,25 @@ def test_run_from_tensorboard(runner, relay_server, user, api, copy_asset):
         assert len(uploaded_files) == 17
 
 
+def test_fetching_artifact_files(user, wandb_init):
+    project = "test"
+    run = wandb_init(entity=user, project=project)
+    artifact = wandb.Artifact("test-artifact", "test-type")
+    with open("boom.txt", "w") as f:
+        f.write("testing")
+    artifact.add_file("boom.txt", "test-name")
+    run.log_artifact(artifact, aliases=["sequence"])
+    artifact.wait()
+    run.finish()
```

Comment:
I think we're encouraging this style, and it's a bit simpler anyway since you don't have to call wait or finish:
```suggestion
    with wandb_init(entity=user, project=project) as run:
        artifact = wandb.Artifact("test-artifact", "test-type")
        with open("boom.txt", "w") as f:
            f.write("testing")
        artifact.add_file("boom.txt", "test-name")
        run.log_artifact(artifact, aliases=["sequence"])
```

Reply:
added!